source/Flotsam.cpp

```diff
@@ -118,6 +118,14 @@ void Flotsam::Move(vector<Visual> &visuals)
 
 
 
+// Pull this flotsam toward a tractor beam.
+void Flotsam::Tractor(Point pull)
+{
+	velocity = pull;
```

Comment:
thinking, the function suggests that this adds to the velocity and not that it gets overwritten, in which case it would be a standard setter function and probably should be named as such